lib/pusher/push_notifications/client.rb

```diff
@@ -40,15 +55,20 @@ def post(resource, payload = {})
       attr_reader :config
       def_delegators :@config, :instance_id, :secret_key
 
-      def build_url(resource)
-        "https://#{instance_id}.#{BASE_URL}/#{instance_id}/#{resource}"
+      def build_publish_url(resource)
+        "https://#{instance_id}.pushnotifications.pusher.com/publish_api/v1/instances/#{instance_id}/#{resource}"
+      end
+
+      def build_users_url(resource)
+        "https://#{instance_id}.pushnotifications.pusher.com/user_api/v1/instances/#{instance_id}/#{resource}"
       end
 
       def headers
         {
           content_type: 'application/json',
           accept: :json,
-          Authorization: "Bearer #{secret_key}"
+          Authorization: "Bearer #{secret_key}",
+          'X-Pusher-Library': "push-notifications-server-ruby #{Pusher::PushNotifications::VERSION}"
```

Comment:
`pusher-push-notifications-ruby <version>`

Reply:
Fixed in 00bfb3c.